### PR TITLE
Update from Sirupsen -> sirupsen

### PR DESCRIPTION
**Pull request by aanm**

### CODE REVIEW

File: common/logging.go [hunk 1]
```diff
@@ -242,24 +241,24 @@ func setupFluentD(logOpts map[string]string, debug bool) {
 // setupLogstash sets up and configures Logstash with the provided options in
 // logOpts. If some options are not provided, sensible defaults are used.
 /// TODO fix me later - needs to be tested with a working logstash setup.
```
Comment (nebril) [inline common/logging.go]: Can we remove it and grab it from git later/track it in a separate issue?

**Review by nebril — COMMENTED** [review]
LGMT, but I think that we shouldn't leave code in comments, but track the fix in separate issue/retrieve the old version to fix from git.

File: common/logging.go (hunk 1 above)
Comment (tgraf) [reply common/logging.go]: What I've started doing is to file a GH issues and refer to it from the comment in the code ie in the form:
```
// GH-1231: text
```

**Comment by aanm** [comment]
@tgraf  @nebril I've left the code commented and wrote with the GH format. If we drop support for logstash we can remove the function later on. TBD in #1578

**Review by ashwinp — APPROVED** [review]
LGTM!

File: NEWS.rst [hunk 2]
```diff
@@ -25,6 +25,10 @@ Documentation
 -------------
 * Add here
 
+Other
+-----
+* Update Sirupsen/logrus to sirupsen/logrus (1573_)
```
Comment (tgraf) [inline NEWS.rst:30]: 👍 Thank you!

**Review by tgraf — APPROVED** [review]

**Pull request merged**